README.md

```diff
@@ -74,8 +74,8 @@ Run these commands:
     meson build
     ninja -C build
 
-On FreeBSD, you need to pass an extra flag to prevent a linking error:
-`meson build -D b_lundef=false`.
+If you use Meson older than 0.48.0 on FreeBSD, you need to pass an extra flag
+to prevent a linking error: `meson build -D b_lundef=false`.
```

Comment:
You should just set the minimum version and remove this from the README entirely. Specifying override_options on meson <0.48.0 will break anyway.

Reply:
Done.